### DISCUSSION TITLE
InfluxDB Cloud

### DISCUSSION THREAD

**Discussion by acabrera2012** [David00/rpi-power-monitor#149] [Q&A]
Apologies, again, if this question has been asked.

When I look at this section of the config it makes sense for a separate install of InfluxDB:

`
[database]
host = "localhost"
port = 8086
username = "root"
password = "password"
database_name = "power_monitor" 
`

But, is it possible to send the power monitor data directly to the InfluxDB Cloud service? [https://www.influxdata.com/products/influxdb-cloud/](url)

**Comment by David00** [comment]
Hey @acabrera2012, I just wanted to respond here and say that I have seen this, and I intend on exploring this and getting back to you soon!

**Reply by acabrera2012** [reply]
Great! Thanks, David.
